dkist/io/asdf/converters/dataset.py

```diff
@@ -27,6 +28,21 @@ def from_yaml_tree(self, node, tag, ctx):
         unit = node.get("unit")
         mask = node.get("mask")
 
+        # If we have a tag older than 1.2.0 then we are going to see if we can
```

Comment:
@braingram Is there a way I can do this by overriding the Tabular1D converter? or is this the "best" available option?

Reply:
Thanks!